src/modules/CRUD/components/Nav/Nav.js

```diff
@@ -54,32 +56,35 @@ export const Nav = ({ getAllLayersAction, resizingMap, layersList, t }) => {
           minimal
         />
       </div>
-      <ul
-        id="CRUD-nav__menu"
-        className="CRUD-nav__menu"
-      >
-        {layersListByGroups.map(({ group, list }) => (
-          <li className="CRUD-nav__group" key={group}>
-            <p className="CRUD-nav__group-name">
-              {group}
-            </p>
-            <ul className="CRUD-nav__list">
-              {list.map(({ name }) => (
-                <li className="CRUD-nav__item" key={name}>
-                  <NavLink
-                    to={generateURI('layer', { layer: name })}
-                  >
-                    <span className="bp3-button bp3-minimal">
-                      <Icon icon="layout" />
-                      <span className="bp3-button-text">{name}</span>
-                    </span>
-                  </NavLink>
-                </li>
-              ))}
-            </ul>
-          </li>
-        ))}
-      </ul>
+      { orderedMenu && (
+        <ul
+          id="CRUD-nav__menu"
```

Comment:
Why this id ?

Reply:
It's for accessibility purpose, 
tldr: it's related to `aria-controls="CRUD-nav__menu"`